osquery/tables/utility/file.cpp

```diff
@@ -33,21 +53,36 @@ const std::map<fs::file_type, std::string> kTypeNames{
     {fs::status_error, "error"},
 };
 
+#endif
+
 void genFileInfo(const fs::path& path,
                  const fs::path& parent,
                  const std::string& pattern,
                  QueryData& results) {
-  // Must provide the path, filename, directory separate from boost path->string
-  // helpers to match any explicit (query-parsed) predicate constraints.
+// Must provide the path, filename, directory separate from boost path->string
+// helpers to match any explicit (query-parsed) predicate constraints.
+
+#if defined(WIN32)
+
+  DWORD dwRtnCode = 0;
+  PSID pSidOwner = NULL;
+  BOOL bRtnBool = TRUE;
+  SID_NAME_USE eUse = SidTypeUnknown;
+  HANDLE hFile;
+  PSECURITY_DESCRIPTOR pSD = NULL;
```

Comment:
nit: prefer `nullptr` if it's possible, again it might not always work for Win API calls.